### ISSUE TITLE
Weird error with current development version

### ISSUE THREAD

**Issue by ddeimeke** [s9y/Serendipity#736]
When I am logged in to my blog, I get the following error when I try to open [this article](https://www.deimeke.net/dirk/blog/index.php?/archives/4008-Onyx-Boox-Max-3.html).

Any ideas?

```
Fatal error: Uncaught ErrorException: Warning: Use of undefined constant ACTIVE_COMMENT_SUBSCRIPTION - assumed 'ACTIVE_COMMENT_SUBSCRIPTION' (this will throw an Error in a future version of PHP) in /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php:409 Stack trace: #0 /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php(409): errorToExceptionHandler(2, 'Use of undefine...', '/srv/www/deimek...', 409, Array) #1 /srv/www/deimeke.net/dirk/blog/include/functions_smarty.inc.php(788): serendipity_printComments(Array, 0, 0, NULL, 'COMMENTS', 'comments.tpl') #2 /srv/www/deimeke.net/dirk/blog/templates_c/2k11/53/99/54/53995459b81e94caa3ed29716a8b93000b4a8aee_0.file.entries.tpl.php(218): serendipity_smarty_printComments(Array, Object(Smarty_Internal_Template)) #3 /srv/www/deimeke.net/dirk/blog/bundled-libs/Smarty/libs/sysplugins/smarty_template_resource_base.php(123): content_5e7f2f00657b00_21644675(Object(Smarty_Internal_Template)) #4 /srv/www/deimeke.net/dirk/blog/bundled-libs/Smarty/libs/sysplugins/sm in /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php on line 409
== ERROR-REPORT (BETA/ALPHA-BUILDS) ==

Fatal Error: Uncaught ErrorException: Warning: Use of undefined constant ACTIVE_COMMENT_SUBSCRIPTION - assumed 'ACTIVE_COMMENT_SUBSCRIPTION' (this will throw an Error in a future version of PHP) in /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php:409 Stack trace: #0 /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php(409): errorToExceptionHandler(2, 'Use of undefine...', '/srv/www/deimek...', 409, Array) #1 /srv/www/deimeke.net/dirk/blog/include/functions_smarty.inc.php(788): serendipity_printComments(Array, 0, 0, NULL, 'COMMENTS', 'comments.tpl') #2 /srv/www/deimeke.net/dirk/blog/templates_c/2k11/53/99/54/53995459b81e94caa3ed29716a8b93000b4a8aee_0.file.entries.tpl.php(218): serendipity_smarty_printComments(Array, Object(Smarty_Internal_Template)) #3 /srv/www/deimeke.net/dirk/blog/bundled-libs/Smarty/libs/sysplugins/smarty_template_resource_base.php(123): content_5e7f2f00657b00_21644675(Object(Smarty_Internal_Template)) #4 /srv/www/deimeke.net/dirk/blog/bundled-libs/Smarty/libs/sysplugins/sm in /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php on line 409.
For more details set $serendipity['production'] = 'debug' in serendipity_config_local.inc.php to receive a stack-trace.



Fatal error:  Uncaught ErrorException: Fatal Error: Uncaught ErrorException: Warning: Use of undefined constant ACTIVE_COMMENT_SUBSCRIPTION - assumed 'ACTIVE_COMMENT_SUBSCRIPTION' (this will throw an Error in a future version of PHP) in /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php:409
Stack trace:
#0 /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php(409): errorToExceptionHandler(2, 'Use of undefine...', '/srv/www/deimek...', 409, Array)
#1 /srv/www/deimeke.net/dirk/blog/include/functions_smarty.inc.php(788): serendipity_printComments(Array, 0, 0, NULL, 'COMMENTS', 'comments.tpl')
#2 /srv/www/deimeke.net/dirk/blog/templates_c/2k11/53/99/54/53995459b81e94caa3ed29716a8b93000b4a8aee_0.file.entries.tpl.php(218): serendipity_smarty_printComments(Array, Object(Smarty_Internal_Template))
#3 /srv/www/deimeke.net/dirk/blog/bundled-libs/Smarty/libs/sysplugins/smarty_template_resource_base.php(123): content_5e7f2f00657b00_21644675(Object(Smarty_Internal_Template))
#4 /srv/www/deimeke.net/dirk/blog/ in /srv/www/deimeke.net/dirk/blog/include/functions_comments.inc.php on line 409
```

**Comment by stephanbrunker** [comment]
That would be my doing. I totally resorted the language constants and nothing _should_ be deleted. But this looks like an accidental deletion. I'll make a hotfix ASAP.

**Comment by stephanbrunker** [comment]
fixed with 712b958 . Apologies for that. There were three constants: ACTIVE_COMMENT_SUBSCRIPTION, PENDING_COMMENT_SUBSCRIPTION and NO_COMMENT_SUBSCRIPTION. The latter two are now obsolete in my development version, but are still recent in current master. The first one is now only used in the admin/comments.tpl and I accidentally deleted that with the other two. I restored all three in the languages where they were translated (en, de, cz, cs and ko), sorted the future obsolete ones under 'orphaned' and 'ACTIVE_' under comments.tpl and synced everything to all the other languages with my new langsorter.

**Comment by ddeimeke** [comment]
Works perfectly fine.
Thanks a lot @stephanbrunker
